packages/trader/src/Modules/Account/Containers/account.jsx

```diff
@@ -91,13 +71,18 @@ class Account extends React.Component {
             if (menu_item.title === 'Verification') {
                 menu_item.is_hidden = !needs_verification;
             }
-            if (menu_item.title === 'Profile') {
-                menu_item.subroutes.forEach(route => {
-                    if (route.path === AppRoutes.financial_assessment) {
-                        route.is_disabled = !is_high_risk_client;
-                    }
-                });
-            }
+            menu_item.subroutes.forEach(route => {
+                if (route.path === AppRoutes.financial_assessment) {
+                    route.is_disabled = !is_high_risk_client;
+                }
+
+                if (
+                    !needs_verification &&
+                    !is_high_risk_client &&
+                    /proof-of-identity|proof-of-address|financial-assessment/.test(route.path)
```

Comment:
may i know what is this actually?

Reply:
@mustofa-binary These are individual checks for the different tabs available that will trigger that specific tab to be disabled.